### ISSUE TITLE
동적 UI widget 내용 변경 문제

### ISSUE THREAD

**Issue by engineer1J** [kookmin-sw/2019-cap1-2019_16#13]
생성까지는 구현하였으나
동적으로 생성된후 내용 변경하는 과정에서 어려움이 있습니다.
블루프린트로 개발중이며 방법을 아는 분은 덧글을 남겨주세요

**Comment by engineer1J** [comment]
UI text의 내용은 배열을 이용해 관리하는 것이 편할 것 같습니다

**Comment by engineer1J** [comment]
현재 timer로 텍스트를 넘기는 것을 구현하였으나 텍스트창의 업데이트와 시스템 내부 integer 값이 맞지 않는 현상이 발견되고 있습니다. 방법을 계속 찾도록 하겠습니다.

**Comment by 610ksh** [comment]
맞습니다~ 유니티에서도 TEXT UI를 관리할때, 배열을 이용해서 처리하는게 좋았던것 같습니다. 언리얼도 마찬가지라 생각합니다. 참고로 배열해서 생각난건데, 언리얼에서도 C++11을 지원하기 때문에 Vector클래스를 적극 이용하는게 좋아보입니다. std::vector와 같은겁니다. 자세한건 찾아보시면 될것 같네요.

**Comment by engineer1J** [comment]
화면 전체에 투명 버튼을 만들어 마우스로 넘기는 방식으로 구현하니 버그가 사라졌습니다. 우선 이 방법으로 채택을 하였습니다.
